src/azul/plugins/metadata/hca/transform.py

```diff
@@ -1399,16 +1399,24 @@ def transform(self) -> Iterable[Contribution]:
         project = self._get_project(self.api_bundle)
 
         matrices = [
-            (file, Submitter.for_file(file))
+            # FIXME: https://github.com/DataBiosphere/azul/issues/3096
+            #        After #3096 retroactively assigns the 'dcp2' submitter ID
+            #        to files produced by DCP/2 Analysis, ensuring all valid
+            #        matrices will have a submitter, store the submitter object
+            #        as the 2nd item in the tuple instead of the category.
+            (file, Submitter.category_for_file(file))
             for file in visitor.files.values()
             if file.is_matrix
         ]
 
         def _matrices_for(category: SubmitterCategory) -> JSONs:
             return [
-                self._matrices(file, submitter)
-                for file, submitter in matrices
-                if submitter.category == category
+                # FIXME: https://github.com/DataBiosphere/azul/issues/3096
```

Comment:
#3096 is already in Approved. It doesn't make sense to add a FIXME referring to an issue that is already fixed.